docroot/modules/custom/sitenow_people/src/Form/SettingsForm.php

```diff
@@ -540,6 +562,11 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       ->save();
     parent::submitForm($form, $form_state);
 
+    $this->configFactory->getEditable(static::SETTINGS)
+      // Save the tag display default.
+      ->set('show_visual_indicators_on_teasers', $show_visual_indicators_on_teasers)
```

Comment:
I prefer more verbose variable and setting names for readability, but if you have any suggestions, I'll take them into account

Reply:
Could be misconstrued as a list or an array instead of essentially a boolean ("show/don't show"). Also, the indicators themselves are not clickable, the entire teaser is.

Reply:
Visual indicator is equally vague about what is being indicated.

It seems like a comment might be able to help provide context? I think there are limits to how clear you can get with verbosely named variables.

Reply:
`show_teaser_link_indicator`?

Reply:
> Moving this back to draft while @quamsta adds in Articles.

@bspeare 

This has been added!

Reply:
Should be resolved https://github.com/uiowa/uiowa/pull/5094/commits/9cabc6ec76ea18ca4f86ecd15312170d56c73fb0